donate/settings.py

```diff
@@ -575,6 +578,9 @@
 STRIPE_API_KEY = env('STRIPE_API_KEY')
 STRIPE_WEBHOOK_SECRET = env('STRIPE_WEBHOOK_SECRET')
 
+# Salesforce
+SALESFORCE_ORGID = env('SALESFORCE_ORGID')
```

Comment:
Does this need to be an env var or can we hardcode it, with an override in the Thunderbird app?

Reply:
env var. Adding a thunderbird app override was more work for no payoff beyond what the env var does for us.